src/modules/35-connectors/pages/createConnectorFromYaml/CreateConnectorFromYamlPage.tsx

```diff
@@ -206,7 +206,7 @@ const CreateConnectorFromYamlPage: React.FC = () => {
                   hideModal()
                 }}
               >
-                {formikProps => (
+                {(formikProps: any) => (
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
                {(formikProps: unknown) => (
```